whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/acl/ResetPersonalObjectAccounting.java

```diff
@@ -17,7 +17,7 @@ public ResetPersonalObjectAccounting(final PersonalObjectAccounting personalObje
     }
 
     @Override
-    @Scheduled(cron = "0 0 0 * * *")
+    @Scheduled(cron = "0 0 0 * * *", zone = EUROPE_AMSTERDAM)
```

Comment:
Does this job need to run on Amsterdam time? the ACL data will be in UTC

Reply:
probably best to have it run in UTC. Updated to this effect